tools/DevDiagnostics/DevHome.DevDiagnostics/Controls/InsightSimpleTextControl.xaml.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.ComponentModel;
+using Microsoft.UI.Xaml.Controls;
+
+namespace DevHome.DevDiagnostics.Controls;
+
+public sealed partial class InsightSimpleTextControl : UserControl
```

Comment:
What additional functionality does this offer beyond a regular TextBox?

Reply:
None, other than being able to apply a set properties to the textbox in XAML rather than programmatically...

Reply:
Does it work with Narrator?

Reply:
Yes.

Reply:
...so why can't you apply those property values to a textbox in xaml?

Reply:
You can... that's the InsightSimpleTextControl.xaml.

This textbox doesn't exist in InsightsPage.xaml. Each insight provides its own user control to represent its insight.

Reply:
sorry to bang on about this, just trying to understand the thinking. Insight.CustomControl is just a UIElement - so why can't the SimpleTextInsight just create a TextBox instead of your custom InsightsSimpleTextControl?

Reply:
Tim explained, makes sense now, thanks.